avocado/utils/software_manager.py

```diff
@@ -517,6 +537,95 @@ def provides(self, name):
         else:
             return None
 
+    def info(self, name):
+        """
+        Returns a name for the given package [name]
```

Comment:
Not sure whether this is suppose to return info about currently installed or available package. It'd be nice to specify it here.

If it's about the installed package than it'd be nice to provide such method for `rpm` based systems (it's a matter of `"rpm -q %s" % name`).

If it's about available package, I'd probably use better name like `query_available_version` or so.

Reply:
Oh, it's about available verison...

Reply:
Actually both, installed or available. I find this quite confusing, what is the target usage of this? I could see a benefit in having `rpm -q $package` as part of API to log installed version, but having a method to see version that is either installed, or not sounds strange to me.

Reply:
worse is when you actually have older package installed, then you get the installed version.